clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerRebalanceListener.java

```diff
@@ -119,15 +119,15 @@
 public interface ConsumerRebalanceListener {
 
     /**
-     * A callback method the user can implement to provide handling of offset commits to a customized store.
+     * A callback method the user can implement to provide handling of offset commits **sent to** a customized store
```

Comment:
Agree with @AndrewJSchofield that it would be an improvement to start stating the general purpose of the callback (cleaning up resources being reassigned.. as Andrew described). Still, I would also keep the offset management example, seems a simple one to understand how to use the callback. I honestly don't know how often it's used out there, but it's one of the [Usage examples](https://github.com/apache/kafka/blob/699438b7f7801115cb7d66dba202b25af7ff8a1f/clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java#L350) described in the `KafkaConsumer` documentation, so from a user's perspective it would align nicely to refer to the same example here I would say.....side note: I may be a bit biased here, I've been recently working on this use case of the callbacks on the new consumer :)